pkg/kube/wait.go

```diff
@@ -160,14 +121,8 @@ func (w *waiter) waitForResources(created ResourceList) error {
 					return false, nil
 				}
 
-			case *corev1.ReplicationController:
-				ok, err = w.podsReadyForObject(value.Namespace, value)
-			case *extensionsv1beta1.ReplicaSet:
-				ok, err = w.podsReadyForObject(value.Namespace, value)
-			case *appsv1beta2.ReplicaSet:
-				ok, err = w.podsReadyForObject(value.Namespace, value)
-			case *appsv1.ReplicaSet:
-				ok, err = w.podsReadyForObject(value.Namespace, value)
+			case *corev1.ReplicationController, *extensionsv1beta1.ReplicaSet, *appsv1beta2.ReplicaSet, *appsv1.ReplicaSet:
+				ok, err = w.podsReadyForObject(v.Namespace, value)
```

Comment:
same here.

Reply:
I don't quite understand, I made these changes based on another part of the code, such as [line 152](https://github.com/helm/helm/blob/cc450563835088945aa79dbfa0bce0bc031fb390/pkg/kube/wait.go#L152) and [line 171](https://github.com/helm/helm/blob/cc450563835088945aa79dbfa0bce0bc031fb390/pkg/kube/wait.go#L171)

Reply:
Referring to this discussion here: https://github.com/helm/helm/pull/6449#discussion_r331755560

This code also uses the unversioned type `v` instead of the versioned type `value`. Please change this to use the versioned type `value`